automatic/kubernetes-cli/kubernetes-cli.json

```diff
@@ -1,10 +1,5 @@
 ﻿{
-    "1.4":  "1.4.12",
-    "1.5":  "1.5.8",
-    "1.6":  "1.6.13",
-    "1.7":  "1.7.16",
-    "1.8":  "1.8.15",
-    "1.9":  "1.9.11",
-    "1.1":  "1.19.5",
-    "1.2":  "1.24.1"
+    "1.22":  "1.22.12",
```

Comment:
No need to include the changes in this file, as AU will update it when it updates the package.

Reply:
I've dropped this files changes. Pleas verify.